ci/rapids/cuda12.2-conda/devcontainer.json

```diff
@@ -0,0 +1,96 @@
+{
+  "image": "rapidsai/devcontainers:24.06-cpp-mambaforge-ubuntu22.04",
```

Comment:
Let's update the devcontainers to 24.08 as well.

Reply:
Done